Doc/library/math.rst

```diff
@@ -206,6 +206,17 @@ Number-theoretic and representation functions
    :meth:`x.__trunc__() <object.__trunc__>`.
 
 
+.. function:: prod(iterable, /, start=1)
+
+   Return the product of a 'start' value (default: 1) times an iterable of numbers.
+
+   When the iterable is empty, return the start value.
+   This function is intended specifically for use with numeric values and may
```

Comment:
I think may is correct here because is indicating that it *may* fail, not that it *should* fail. *should* also sounds like an untested expectation. Notice that the docstring of `sum` also uses may.

Reply:
Hmm for me _should_ is more a obligation (I am not English language native). But reading https://www.ietf.org/rfc/rfc2119.txt the _must_ word can be used here. BTW, is just an opinion. may can be used too